cvmfs/cache_ram.h

```diff
@@ -13,75 +13,275 @@
 
 #include "cache.h"
 #include "hash.h"
+#include "kvstore.h"
+#include "statistics.h"
 #include "util/pointer.h"
 
 namespace cache {
 
+// The null hash (hashed output is all null bytes) serves as a marker for
+// an invalid handle
+static const shash::Any kInvalidHandle;
+
+static const unsigned kMaxHandles = 8192;
```

Comment:
We probably would like to have this configurable, but let's move this to another PR.